content/blog/2021-07-03-darktable-3.6/index.md

```diff
@@ -10,21 +10,21 @@ Translations of this article: [German](https://www.bilddateien.de/blog/2021-07-0
 
 The darktable team is proud to announce our second summer feature release, darktable 3.6. Merry (summer) Christmas!
 
-This is the first of two release this year and, from here on in, we intend to issue two new feature releases each year, around the summer and winter solstices.
+This is the first of two releases this year and, from here on, we intend to issue two new feature releases each year, around the summer and winter solstices.
 
 As usual, all of the new features are fully documented in the [user manual](https://darktable.org/usermanual/3.6/en). The manual is still English-only for now, but translations are in progress ([here](https://darktable-org.github.io/dtdocs/)) and an epub version is now available.
 
 ## Module Groups & Quick Access Panel
 
 ### Quick access panel 
 
-The *quick access panel* is a replacement for *basic adjustments* and then some. This feature provides a new interface to the existing processing modules within a single unified layout. Users can add controls from _any_ module to the quick access panel for increased productivity and enhanced ergonomics. 
+The *quick access panel* is a replacement for *basic adjustments*. This feature provides a new interface to the existing processing modules within a single unified layout. Users can add controls from _any_ module to the quick access panel for increased productivity and enhanced ergonomics. 
```

Comment:
I would prefer to change this rather than remove it. The point is that it's a lot more than just a replacement for basic adjustments.

Reply:
so it's not a replacement, it is a new featre that supercedes the needfor a basicadjustments module.

Reply:
yes, though replacing basic adjustments was the original motivation

Reply:
May it's just my limited knowledge of the English language...: The sentence "The *quick access panel* is a replacement for *basic adjustments* __and then some__." didn't sound like a proper sentence to me. I thought the words "and then some" were some forgotten leftovers from a refactoring. The meaning wasn't clear to me. Maybe you have an idea for easier to understand rephrasing?

Reply:
It means "and much more". It's more an Americanism - I've been watching too much netflix.

Reply:
I then suggest to simply write it as "The *quick access panel* is a replacement for *basic adjustments* and much more". Would that be good? Only question remaining to me, how to edit my PR to make that change...?!? :-/

Reply:
yes, "and much more" reads better. The expression "and then some" is correct English, but a bit casual for a user guide.